examples/stripe-integration/app/routes/buy.tsx

```diff
@@ -0,0 +1,37 @@
+import { Form, redirect, useLoaderData } from "remix";
+import type { ActionFunction } from "remix";
+import { getStripeSession, getDomainUrl } from "~/utils/stripe.server";
+
+type loaderData = {
+  ENV: {
+    PRICE_ID: string;
+  };
+};
+export const loader = async (): Promise<loaderData> => {
+  return {
+    ENV: {
+      PRICE_ID: process.env.PRICE_ID as string
+    }
+  };
```

Comment:
Be sure to return using the `json` function if you're not returning a `Response`

```suggestion
  return json({
    ENV: {
      PRICE_ID: process.env.PRICE_ID as string
    }
  });
```